src/lib/Runner.ts

```diff
@@ -269,10 +269,15 @@ export class Runner {
             sourceBranch: pullRequest.sourceBranch,
             targetBranch: pullRequest.targetBranch,
           });
-          await landRequest.setStatus(
-            'fail',
-            `Unable to merge pull request ${result.reason ? `: ${result.reason}` : ''}`,
-          );
+
+          const hasErrors = result.reason && result.reason.error;
+          const mergeCheckErrors =
+            hasErrors && result.reason.error.fields
+              ? ' ' + result.reason.error.fields.merge_checks.join(', ')
```

Comment:
we could do `.join('\n')` so that UI does not break for lengthy strings.

Reply:
ah that doesn't work :( I think because we're using spans... but i've made some css updates to account for lengthy strings